MIGRATION.md

```diff
@@ -3,6 +3,10 @@
 API changes in MoveIt releases
 
 ## ROS Noetic
+- Added method `maximumRotation` to `RobotState`.
+- The method `distanceRotation` is now a virtual method of `JointModel` and implemented for all joint models.
+- Modified `CollisionEnvBullet`. It now takes a parameter `max_rotation_per_step_`, and subdivides CCD requests into sub-steps to better approximate rotations
+- Renamed `addAttachedOjects` to `createAttachedCollisionObjects` in `CollisionEnvBullet` to reflect the non-mutating nature of the method.
```

Comment:
I'm not sure this should be mentioned either. It's an API change, but it's not user-facing unless users hack in the deep internals for their own projects. Usually only `collision_detection` interfaces should be used.